test/system/issues_test.rb

```diff
@@ -158,7 +158,35 @@ def test_issue_index_with_multiple_roles
 
     visit issues_path
 
-    assert_link I18n.t("issues.index.reports_count", :count => issue1.reports_count), :href => issue_path(issue1)
-    assert_link I18n.t("issues.index.reports_count", :count => issue2.reports_count), :href => issue_path(issue2)
+    assert_link I18n.t("issues.page.reports_count", :count => issue1.reports_count), :href => issue_path(issue1)
+    assert_link I18n.t("issues.page.reports_count", :count => issue2.reports_count), :href => issue_path(issue2)
+  end
+
+  def test_issues_pagination
+    1.upto(80).each do
+      user = create(:user)
+      create(:issue, :reportable => user, :reported_user => user, :assigned_role => "administrator")
+    end
+
+    sign_in_as(create(:administrator_user))
+
+    visit issues_path
+
+    # First Page
+    assert_no_content I18n.t("issues.index.user_not_found")
+    assert_no_content I18n.t("issues.index.issues_not_found")
+    assert_css "tr", :count => 51
+
+    # Second Page
+    click_on I18n.t("issues.page.older_issues")
+    assert_no_content I18n.t("issues.index.user_not_found")
+    assert_no_content I18n.t("issues.index.issues_not_found")
+    assert_css "tr", :count => 31, :wait => 1.5
```

Comment:
I assume that `:wait` is a timeout? Why do we need that - what happens if we don't have it? Is there a default timeout that is too short or does it wait forever?

Reply:
That doesn't really make any sense - a shorter timeout shouldn't fix it. I suspect it was actually just chance that you set a timeout and then that run happened to work within the timeout.

All of which is why I hat tests which rely on timeouts, because they're not deterministic and can fail arbitrarily on different machines or even on the same machine under different conditions.

It seems it's unavoidable with browser based testing like this though, so I suspect we should just increase the default to 5 or 10 or something.

Reply:
I agree with you, it seems strange. I thought, maybe default time of my local machine was different with the one of the versions of GitHub workflow tests. I'll remove it and push without `:wait`-s. Let's see if it still fails.

Reply:
All tests have passed. It's strange because in the first version all the tests were failing. Maybe changing a solution helped timing.